### ISSUE TITLE
Doctest failure for universum-1.0.4

### ISSUE THREAD

**Issue by DanBurton** [serokell/universum#127]
As seen on the Stackage build server:

```
> /tmp/stackage-build12/universum-1.0.4$ dist/build/universum-doctest/universum-doctest
### Failure in src/Universum/String/Conversion.hs:78: expression `decodeUtf8 @Text @ByteString "\208\208\176\209\130\208\176\208\186"'
expected: "\65533\65533\1090\1072\1082"
 but got: "\65533\1072\1090\1072\1082"
Examples: 112  Tried: 111  Errors: 0  Failures: 1
```

**Comment by ChShersh** [comment]
@DanBurton Thanks for notification! I will look into this. Travis CI is passes. The problem is in compatibility with new version of `text` package. Stackage nightly has `text-1.2.3.0` while LTS-10.3 has `text-1.2.2.2`.

**Comment by DanBurton** [comment]
Yes, I do believe `text-1.2.3.0` is the issue.

I was able to reproduce the test failure on my local machine like so:

```bash
$ stack unpack universum-1.0.4
$ cd universum-1.0.4
$ echo 'resolver: nightly-2018-01-26' > stack.yaml
$ stack test
```

This reported the same failed test case as above.

When I tried with nightly-2018-01-25, it passed. Here's [the diff](https://www.stackage.org/diff/nightly-2018-01-25/nightly-2018-01-26) between those nightlies.

**Comment by ChShersh** [comment]
@DanBurton This should be fixed in `universum-1.0.4.1`: 

* https://hackage.haskell.org/package/universum-1.0.4.1
